### PR TITLE
Handles recommended and fixable flags in rules documentation

### PR DESCRIPTION
**Pull request by weirdpattern**
Reformat the list of rules into a table
Add recommended and fixable flags
Update dependencies
Update .babelrc to node: 6

Output will look like...

Not sure if you see a benefit in adding a flag to either render the old look vs the new look

closes #32
closes #33
closes #34

### CODE REVIEW

File: src/actions/update-readme.js [hunk 1]
```diff
@@ -32,20 +32,24 @@ exports = module.exports = (readme, ruleMeta, { pluginName }) => {
...
-  const ruleLines = meta
+  const header = '| Name | Description | :heavy_check_mark: | :wrench: |'
+  const delimiter = '| ----- | ----- | ----- | ----- |'
+
+  const rulesTable = meta
     .map(
-      ({ name, description, extraDescription }) =>
-        `* [\`${pluginName}/${name}\`](./docs/rules/${name}.md) \u{2014} ${exports.handleDescription(
+      ({ name, description, extraDescription, recommended, fixable }) =>
+        `| [${pluginName}/${name}](./docs/rules/${name}.md) | ${exports.handleDescription(
```
Comment (j-f1) [inline src/actions/update-readme.js]: Can you switch this back to code format?
```suggestion
        `| [\`${pluginName}/${name}\`](./docs/rules/${name}.md) | ${exports.handleDescription(
```

File: package-lock.json [hunk 2]
```diff
@@ -5,151 +5,37 @@
   "requires": true,
```
Comment (j-f1) [inline package-lock.json]: You can delete this file if you’d like — I prefer Yarn now.

File: src/actions/update-readme.js [hunk 3]
```diff
@@ -32,20 +32,24 @@ exports = module.exports = (readme, ruleMeta, { pluginName }) => {
...
-        )}`
+        )} | ${recommended ? ':heavy_check_mark:' : ''} | ${
+          fixable != null ? ':wrench:' : ''
+        } |`
     )
     .join(nl)
 
-  return `<!-- begin rule list -->${nl}${ruleLines}${nl}<!-- end rule list -->
-    `.trim()
+  return `<!-- begin rule list -->${nl}${header}${nl}${delimiter}${nl}${rulesTable}${nl}<!-- end rule list -->`.trim()
```
Comment (j-f1) [inline src/actions/update-readme.js]:
```suggestion
  return `
<!-- begin rule list -->
**Key**: :heavy_check_mark: = recommended, :wrench: = fixable
| Name | Description | :heavy_check_mark: | :wrench: |
| ---- | ----------- | ------------------ | -------- |
${rulesTable}
<!-- end rule list -->
  `.trim().replace(/\n/g, nl)
```

File: package.json [hunk 4]
```diff
@@ -61,6 +62,6 @@
     "mz": "^2.7.0",
     "ora": "^3.0.0",
     "read-pkg-up": "^4.0.0",
-    "regenerator-runtime": "^0.12.1"
+    "regenerator-runtime": "^0.13.1"
```
Comment (j-f1) [inline package.json]: Can this be removed since Node 6 supports generators natively?

File: src/actions/update-readme.js (hunk 3 above)
Comment (weirdpattern) [reply src/actions/update-readme.js]: I thought of adding this legend here vs hardcoding it directly in the `readme.md`... I think you are right, we simply cannot force users into remember adding this each time...

I will replace this code with your suggestion.

File: package.json (hunk 4 above)
Comment (weirdpattern) [reply package.json]: Sure thing

File: package-lock.json (hunk 2 above)
Comment (j-f1) [reply package-lock.json]: Sure, go ahead!

**Comment by weirdpattern** [comment]
Alright, we should be ready to go

**Comment by weirdpattern** [comment]
Ok missed the last comment.. working on this

**Comment by j-f1** [comment]
Published as `v0.2.0` 🎉

**Pull request merged**